src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -347,7 +347,7 @@ file that was distributed with this source code.
 
 {% block form_row %}
     {% set show_label = show_label|default(true) %}
-    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}" id="sonata-ba-field-container-{{ id }}">
+    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}{% if attr.row_class|default('') %} {{ attr.row_class|default('') }}{% endif %}" id="sonata-ba-field-container-{{ id }}">
```

Comment:
```suggestion
    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}{{ attr.row_class|default('') }}" id="sonata-ba-field-container-{{ id }}">
```

Reply:
you forgot about a break before `attr.row_class`

Reply:
@kunicmarko20 doesn't work because need whitespace before `attr.row_class`

Reply:
Can't we just add a space between them?

Reply:
I think yes. 
Result:
```html
<div class="form-group " id="sonata-ba-field-container-name">
```
whitespace before `"`